x-pack/plugins/spaces/server/routes/api/external/disable_legacy_url_aliases.ts

```diff
@@ -20,15 +20,25 @@ export function initDisableLegacyUrlAliasesApi(deps: ExternalRouteDeps) {
       path: '/api/spaces/_disable_legacy_url_aliases',
       options: {
         access: isServerless ? 'internal' : 'public',
-        description: `Disable legacy URL aliases`,
+        summary: `Disable legacy URL aliases`,
+        tags: ['oas-tag:spaces'],
       },
       validate: {
         body: schema.object({
           aliases: schema.arrayOf(
             schema.object({
-              targetSpace: schema.string(),
-              targetType: schema.string(),
-              sourceId: schema.string(),
+              targetSpace: schema.string({
+                meta: { description: 'The space where the alias target object exists.' },
+              }),
+              targetType: schema.string({
+                meta: { description: 'The type of alias target object. ' },
+              }),
+              sourceId: schema.string({
+                meta: {
+                  description:
+                    'The alias source object identifier. This is the legacy object identifier.',
+                },
+              }),
             })
```

Comment:
> [!NOTE]  
> Add response description for 204 status code as in the original doc.

Reply:
I only know how to do that in the "router.versioned" cases. When I tried adding request and response bodies to the APIs like this one that just use "router", the validation failed. Perhaps the core team can provide an example if this isn't something you know how to add either.